packages/components/table/src/table-body/render-helper.ts

```diff
@@ -78,7 +78,7 @@ function useRender<T>(props: Partial<TableBodyProps<T>>) {
         if (!rowspan || !colspan) {
           return null
         }
-        const columnData = { ...column }
+        const columnData = Object.assign({}, column)
```

Comment:
It is supported, but it can be deconstructed for the first time in this loop, but the object in the following loop becomes an empty object. I didn't say that it is incompatible, but in the original writing method, I don't know what causes it to become an empty object in version 72. You can avoid this situation by changing it to Object.assign. You can try the bug behavior of table in version 72 yourself.